lib/rules/comma-dangle.js

```diff
@@ -81,49 +81,49 @@ module.exports = {
             category: "Stylistic Issues",
             recommended: false
         },
-
         fixable: "code",
-
-        schema: [
-            {
-                defs: {
-                    value: {
-                        enum: [
-                            "always",
-                            "always-multiline",
-                            "only-multiline",
-                            "never"
-                        ]
-                    },
-                    valueWithIgnore: {
-                        anyOf: [
-                            {
-                                $ref: "#/defs/value"
```

Comment:
That makes sense, thanks for explaining. I agree that option 2 seems safest, if it's not too difficult a change. Maybe we can switch to option 1 in a future major release.